src/components/FilterPanel.js

```diff
@@ -221,9 +221,9 @@ export default function FilterPanel(props) {
                 if (d) {
                     const zipValid = zipCodeFilter.zipCode.match(/\d{5}/);
                     if (zipValid) {
-                        setSortBy("miles");
                         const myCoordinates = usZips[zipCodeFilter.zipCode];
                         if (myCoordinates) {
+                            setSortBy("miles");
```

Comment:
I know this is out of scope of this change, but:

 `geolib`'s `getDistance()` function says it's 3rd arg is "accuracy (in meters). By default the accuracy is 1 meter. … You can set it higher to have the result rounded to the next value that is divisible by your chosen accuracy."

~Why are we rounding ourselves instead of using the built-in rounding function?~

OK, we're not rounding we're _converting_.

Why are we converting ourselves with a hardcoded physical constant (what if the earth accelerates and we redefine the relationship between meters and feet because the US is left behind the metric system?) instead of using `geolib.convertDistance(value, 'mi')`?

Reply:
I didn't see "convertDistance"... good idea.  I still have to round the miles though because the accuracy is in meters.